.github/workflows/releaser.yaml

```diff
@@ -6,11 +6,21 @@ on:
     - cron: '30 9 * * 3'
 
   # allow manual triggering of the workflow by a maintainer with no inputs
-  workflow_dispatch: {}
+  workflow_dispatch:
+    inputs:
+      release_type:
+        description: "Type of the release"
+        type: choice
+        required: true
+        options:
+          - 'patch'
+          - 'minor'
+          - 'major'
 
 jobs:
   releaser:
     uses: signoz/primus.workflows/.github/workflows/releaser-signoz.yaml@main
     secrets: inherit
     with:
       PRIMUS_REF: main
+      RELEASE_TYPE: ${{ inputs.release_type || 'minor' }}
```

Comment:
No need to give a default here. The choices are mandatory.

Reply:
This is fallback that will be used by scheduled cron.